utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -529,13 +807,85 @@ inline void remove_substring(string &str, string &pattern)
    }
 }
 
+// compares the output of PKD3-PKD3 variants
+void compare_pkd(Rpp8u* output, Rpp8u* refOutput, RpptDescPtr dstDescPtr, RpptImagePatch *dstImgSizes, int refOutputHeight, int refOutputWidth, int refOutputSize, int &fileMatch)
```

Comment:
change the naming to compare_outputs_pkd
similar for PLN function

Reply:
Done